build.toml

```diff
@@ -153,3 +153,10 @@ payload_dir = 'Source\Built\Scripting Examples'
 install_destination = 'ni-paths-LV{veristand_version}DIR\examples\NI VeriStand Custom Devices\Routing and Faulting'
 control_file = 'control_examples'
 package_output_dir = 'Source\Built'
+
+[[package]]
+type = 'nipkg'
+payload_dir = 'Source\Shared\Custom Device LabVIEW Common'
```

Comment:
@buckd are you okay with `payload_dir` being set somewhere outside of `Source/Built`?

Reply:
Yes, payload_dir can be anywhere, it just happens to be some `Built` directory for most of our existing repos.